src/lsm/groove.zig

```diff
@@ -397,33 +420,21 @@ pub fn GrooveType(
         /// sufficient to query this hashmap alone to know the state of the LSM trees.
         prefetch_objects: PrefetchObjects,
 
+        pub const IndexTreesOptions = IndexTreesOptions;
```

Comment:
Is there a shadowing issue here with both names the same? Or am I seeing double?

Reply:
Nope, this is dead code :facepalm:. Good eye!